scipy/interpolate/_interpolate.py

```diff
@@ -795,8 +797,7 @@ def __call__(self, x, nu=0, extrapolate=None):
 
 
 class PPoly(_PPolyBase):
-    """
-    Piecewise polynomial in terms of coefficients and breakpoints
+    """Piecewise interpolating polynomial in the power basis.
```

Comment:
```suggestion
    """Piecewise polynomial in the power basis.
```

This  is just a representation of a piecewise polynomial. No data to interpolate (that would be its subclasses, CubicSpline et al)